src/backend/aspen/api/utils/phylo.py

```diff
@@ -208,6 +287,7 @@ async def process_phylo_tree(
     json_data = json.loads(data)
 
     if id_style == "public":
+        json_data = await _set_colors(db, json_data, phylo_run)
```

Comment:
modify color when we download trees with public IDs as well. thanks for the tip Jess!!